common/depot/helpers.go

```diff
@@ -0,0 +1,21 @@
+package depot
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	depotReposPath = "/repos"
+)
+
+var (
+	httpClient = &http.Client{Timeout: 10 * time.Second}
```

Comment:
Bruh. I know its declared where I put the comment. Where is it _used_. I couldn't see it used anywhere in this file.